cpg_pipes/types.py

```diff
@@ -93,9 +93,11 @@ class CramPath(AlignmentInput):
     """
 
     def __init__(
-        self, 
-        path: str | Path, 
+        self,
+        path: str | Path,
         index_path: Path | str | None = None,
+        # TODO: should this be required?
```

Comment:
I think it would be cleaner if we always require this, so we don't assume that the assembly used for realignment is identical to the one required for decoding the CRAM.

Does that mean we'd need to backfill the corresponding fields in the sample metadata server for our existing CRAMs though?